test/RunLibs.hx

```diff
@@ -0,0 +1,11 @@
+function main() {
+    Test.run('_generated-libs',"",[
+        'node',
+        'three',
```

Comment:
Would be useful to also do three orbit controls in the same dts2hx execution (see run lib sh)